custom_components/example_light/light.py

```diff
@@ -1,14 +1,18 @@
 """Platform for light integration."""
+from __future__ import annotations
+
 import logging
 
 import awesomelights
-import voluptuous as vol
-
 import homeassistant.helpers.config_validation as cv
+import voluptuous as vol
```

Comment:
Please group the imports according to PEP8. Isort will do this automatically. This voluptuous import is a 3rr party import and should be grouped with awesomelights.